pkg/scheduler/factory/factory.go

```diff
@@ -1076,24 +1076,15 @@ func NewPodInformer(client clientset.Interface, resyncPeriod time.Duration) core
 
 func (c *configFactory) MakeDefaultErrorFunc(backoff *util.PodBackoff, podQueue internalqueue.SchedulingQueue) func(pod *v1.Pod, err error) {
```

Comment:
@misterikkit  what is the advantage of this part ?

Reply:
@krmayankk The goal of phase :panda_face: is to break the dependency on the `configFactory` type. At the end of this mini-phase, we should be able to construct a `Scheduler` instance without ever instantiating a `configFactory`.

I'm not concerned with moving the code out of  `k8s.io/kubernetes/pkg/scheduler/factory` at this time. Once the code exists as a bunch of regular functions, it will be much easier to move it around.

Reply:
thanks from the issue it was not clear, makes sense.

Reply:
yeah I wrote about it in https://github.com/kubernetes/kubernetes/issues/72547 but didn't broadcast that very well